docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -704,7 +704,7 @@ The `EventBus` object provides methods to:
 [source, java]
 ----
 // Case 1
-bus.sendAndForget("greeting", name)
+bus.send("greeting", name)
```

Comment:
I know that there is the `io.vertx.mutiny.core.eventbus.EventBus.requestAndForget(String, Object, DeliveryOptions)` method which has a little cleaner name but it registers a reply handler internally which is probably less efficient compared to `send()` so :shrug: ...